### PR TITLE
JJDZR12IL-17

### PR DESCRIPTION
**Pull request by Ne0rin**
Added new project WebApp (Spring Web and Thymeleaf). Initialization new controllers.

### CODE REVIEW

**Review by olszewskiszym — APPROVED** [review]
Ready for merge.

File: WebApp/src/main/java/com/isa/webapp/controller/MainController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.isa.webapp.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class MainController {
+
+    @GetMapping("/")
+    public String getHomePage () {
```
Comment (malik-maciej) [inline WebApp/src/main/java/com/isa/webapp/controller/MainController.java]: formatowanie ;)

File: WebApp/src/main/java/com/isa/webapp/controller/MainController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        return "index";
+    }
+
+    @GetMapping("/login")
+    public String getLoginaPage () {
```
Comment (malik-maciej) [inline WebApp/src/main/java/com/isa/webapp/controller/MainController.java]: `Login`

File: WebApp/src/main/resources/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+ <h1>Login Page</h1>
+ <nav>
+     <a th:href="@{/}">Main page</a> | <a th:href="@{/login}">Login</a> | <a th:href="@{/registration}">Registration</a>
+ </nav>
```
Comment (malik-maciej) [inline WebApp/src/main/resources/templates/login.html]: formatowanie

**Review by malik-maciej — APPROVED** [review]

**Pull request merged**